README.md

```diff
@@ -93,8 +102,33 @@ python setup.py install
 MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py install
 ```
 
-If while building from within an anaconda environment you come across errors similar to the following:
+Alternatively, the build process can build SoX (and codecs such as libmad, lame and flac) statically and torchaudio can link them, by defining `BUILD_SOX` environment variable.
+The build process will fetch and build SoX, liblame, libmad, flac before building extension.
+
+```bash
+# Linux
+BUILD_SOX= python setup.py install
 
+# OSX
+BUILD_SOX= MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py install
+```
+
+This is known to work on a few major Linux distributions such as Ubuntu and CentOS 7 and macOS.
```

Comment:
nit: This is known to work on linux and unix distributions such as Ubuntu and CentOS 7 and macOS.